README.md

```diff
@@ -131,6 +131,8 @@ They exist to document what may be implemented by [Nostr](https://github.com/nos
 | `1971`        | Problem Tracker                 | [nostrocket][nostrocket]               |
 | `1984`        | Reporting                       | [56](56.md)                            |
 | `1985`        | Label                           | [32](32.md)                            |
+| `1986`        | Relay reviews                   | [NIP-85]                               |
```

Comment:
I don't know if it is appropriate to add NIP kind that is not yet merged.

Reply:
I don't mind, if you change the link. You could link to the PR, but it is also called NIP-85.

Reply:
I agree, but this makes PRs second class citizens compared to wikifreedia entriea. Maybe we should have a "unmerges and third party" kinds table?

Reply:
For now, fiatjaf's changes look good.